base/sort.jl

```diff
@@ -811,6 +811,14 @@ Characteristics:
 * *quadratic performance* in the number of elements to be sorted:
 it is well-suited to small collections but should not be used for large ones.
 """
+
+
+"""
+    SMALL_ALGORITHM
+
+Default algorithm for small arreys
+
+"""
 const InsertionSort = InsertionSortAlg()
 const SMALL_ALGORITHM = InsertionSortAlg()
```

Comment:
This is a much better docstring, thanks.

```suggestion
const InsertionSort = InsertionSortAlg()

"""
    SMALL_ALGORITHM

Default algorithm for small arrays
"""
const SMALL_ALGORITHM = InsertionSortAlg()
```

The docstring should be directly above the definition of SMALL_ALGORITHM so that it applies to that const, rather than the InsertionSort const (which already has a docstring).